Dev/Gitlab/gitlab_merge_requests.js

```diff
@@ -0,0 +1,433 @@
+
+function request(path) {
+  const httpTransport = require('https');
+  const responseEncoding = 'utf8';
+  const httpOptions = {
+    hostname: gitlab_domain,
+    port: '443',
+    path,
+    method: 'GET',
+    headers: { 'PRIVATE-TOKEN': private_token }
+  };
+  httpOptions.headers['User-Agent'] =
+    'bitbar/gitlab_projects - node ' + process.version;
+
+  return new Promise((resolve, reject) => {
+    const request = httpTransport
+      .request(httpOptions, (res) => {
+        let responseBufs = [];
+        let responseStr = '';
+
+        res
+          .on('data', (chunk) => {
+            if (Buffer.isBuffer(chunk)) {
+              responseBufs.push(chunk);
+            } else {
+              responseStr = responseStr + chunk;
+            }
+          })
+          .on('end', () => {
+            responseStr =
+              responseBufs.length > 0
+                ? Buffer.concat(responseBufs).toString(responseEncoding)
```

Comment:
line 132, col 17, Misleading line break before '?'; readers may interpret this as an expression boundary.